src/operator/find.ts

```diff
@@ -85,6 +85,7 @@ export class FindValueSubscriber<T> extends Subscriber<T> {
 
     destination.next(value);
     destination.complete();
```

Comment:
This seems odd... calling `complete` on the destination, should end up calling `unsubscribe` on the destination, and subsequently up the chain to here. Although maybe I've got my head in a knot thinking about it. cc @trxcllnt